_includes/codeTable.html

```diff
@@ -196,22 +201,22 @@
         <th>{{ site.data.i18n.form.presets.date.labels.created[page.lang] }}</th>
       </tr>
     </thead>
-    <tbody>
+    <tbody class="row wb-eqht">
       {%- for govLevels in site.data.code -%}
         {%- assign govLevel = govLevels[0] -%}
         {%- for admin in govLevels[1] -%}
           {%- for release in admin[1].releases -%}
           {%- unless release.tags.en contains site.excludeTag -%}
-          <tr>
+          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;" role="row">
```

Comment:
Think this is from @smellems comment
```suggestion
          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;">
```